docs/economics/incentives.rst

```diff
@@ -172,7 +172,7 @@ A range of external factors also have to be considered in the incentives for the
 
 - **Exchange rate fluctuations**. Vaults have a risk of having their `COL` liquidated if the `COL/BTC` exchange rate drops below the :ref:`liquidationThreshold`. In this case, the collateral is liquidated as described in :ref:`liquidations`. Liquidations describe that users can restore the `interBTC` to `BTC` peg by burning `interBTC` for `COL`. However, in a continuous drop of the exchange rate the value of `COL` will fall below the value of the burned `interBTC`. As such, the system relies on actors that execute fast arbitrage trades of `interBTC` for `COL`.
 
-- **Counterparty risk for BTC in custody**. When a user locks BTC with the Vault, he implicitly sells a BTC call option to the Vault. The Vault can, at any point in time, decide to exercise this option by "stealing" the user's BTC. The price for this option is determined by *spot_price + punishment_fee* (*punishment_fee* is essentially the option premium). The main issue here is that we do not know how to price this option, because it has no expiry date - so this deal between the User and the Vault essentially becomes a BTC perpetual that can be physically exercised at any point in time (American-style).
+- **Counterparty risk for BTC in custody**. When a user locks BTC with the Vault, they implicitly sells a BTC call option to the Vault. The Vault can, at any point in time, decide to exercise this option by "stealing" the user's BTC. The price for this option is determined by *spot_price + punishment_fee* (*punishment_fee* is essentially the option premium). The main issue here is that we do not know how to price this option, because it has no expiry date - so this deal between the User and the Vault essentially becomes a BTC perpetual that can be physically exercised at any point in time (American-style).
```

Comment:
```suggestion
- **Counterparty risk for BTC in custody**. When a user locks BTC with the Vault, they implicitly sell a BTC call option to the Vault. The Vault can, at any point in time, decide to exercise this option by "stealing" the user's BTC. The price for this option is determined by *spot_price + punishment_fee* (*punishment_fee* is essentially the option premium). The main issue here is that we do not know how to price this option, because it has no expiry date - so this deal between the User and the Vault essentially becomes a BTC perpetual that can be physically exercised at any point in time (American-style).
```